scripts/xnat/t1_qa_functions.py

```diff
@@ -154,4 +157,7 @@ def process_phantom_session( interface, project, subject, session, label, force_
             run_phantom_qa( interface, project, subject, session, label, dicom_path )
         else:
             # If there was no matching scan in the session, print a warning
-            print "WARNING: ADNI phantom session %s/%s does not have a usable T1-weighted scan" % (session,label)
+            warning = "WARNING: ADNI phantom session: %s, experiment: %s, subject: %s does not have \
+                       a usable T1-weighted scan" % (session, experiment, subject)
```

Comment:
@sbenito we've converted to using the `.format()` of formatting strings in python. Can you update to using:

```
warning = "WARNING: ADNI phantom session: {}, experiment: {}, subject: {} does not have \
                   a usable T1-weighted scan".format(session, experiment, subject)
```